package.json

```diff
@@ -11,7 +11,7 @@
     "test-electron": "electron test/electron.js",
     "test-prebuild": "cross-env PREBUILDS_ONLY=1 npm t",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "rebuild": "node-gyp rebuild",
+    "rebuild": "npm run install --build-from-source",
```

Comment:
Small unrelated change; previously `npm run rebuild` would not use the locally-installed `node-gyp`.